### ISSUE TITLE
Error in plotFastqcPCA()

### ISSUE THREAD

**Issue by steveped** [UofABioinformaticsHub/ngsReports#19]
Hey Chris. 

I get an error when setting `usePlotly = TRUE`. It's consistent across all datasets, including the example dataset
```
plotFastqcPCA(fdl,  module = "Per_sequence_quality_scores", clusterType = "colour", cluster = TRUE, usePlotly = TRUE)
 Error in split.default(x = seq_len(nrow(x)), f = f, drop = drop, ...) : 
  group length is 0 but data length > 0 
```

**Comment by steveped** [comment]
Once this is fixed I'll merge into master, push to Bioc-devel & resubmit the paper

**Comment by steveped** [comment]
Can we close this?

**Comment by CMWbio** [comment]
yes, fixed in 3898b7b4c19dc96fa95ce999792bbd3f04d50b0f
